lib/crypto/test/crypto_SUITE.erl

```diff
@@ -1008,7 +1027,8 @@ group_config(rsa = Type, Config) ->
                   rsa_oaep(),
                   no_padding()
                  ],
-    [{sign_verify, SignVerify}, {pub_priv_encrypt, PubPrivEnc} | Config];
+    Generate = [{rsa, 1024, 3}, {rsa, 2048, 17}, {rsa, 3072, 65537}],
```

Comment:
The generate/1 test case fails on FIPS 140-2 enabled libs for the 1024 length keys.  See the `case ?config(fips, Config) of ... end` a few lines above for inspiration of a fix.  I haven't found any clear indication anywhere whether the 2k length is enough, but the 1k is surely to short in the FIPS case.